Lib/email/utils.py

```diff
@@ -81,7 +81,7 @@ def formataddr(pair, charset='utf-8'):
     If the first element of pair is false, then the second element is
     returned unmodified.
 
-    Optional charset if given is the character set that is used to encode
+    Optional charset, if given, is the character set that is used to encode
```

Comment:
I agree to this change. IMHO is more natural read this, but I don't know if this change is normal for a native English speaker.

But, I think that we should remove the `if given` sentences, because, when the docstring say "Optional charset"  that reference that charset could be given or not. Optional params.

Reply:
Thanks for your reply. I am happy about any change which makes the docstring more readable.

And indeed, the sentence is much nicer without the `if given` clause.

I applied your suggestion with a follow up commit.

I noticed this `if given` pattern is used more often in this module. Should the other places also be updated?